packages/demo/.storybook/main.js

```diff
@@ -0,0 +1,48 @@
+const path = require('path');
+const { lstatSync, readdirSync } = require('fs');
+
+// find @superset-ui packages
+const basePath = path.resolve(__dirname, '../../../node_modules/@superset-ui');
+const packages = readdirSync(basePath).filter(name => {
+  const stat = lstatSync(path.join(basePath, name));
+  return stat.isSymbolicLink();
+});
+
+module.exports = {
+  addons: [
+    '@storybook/preset-typescript',
+    '@storybook/addon-actions/register',
+    '@storybook/addon-knobs/register',
+    'storybook-addon-jsx/register',
+    '@storybook/addon-links/register',
+  ],
+  stories: [
+    '../storybook/stories/**/*Stories.[tj]sx',
+  ],
```

Comment:
So we don't have to manually import each stories module. Might worth consider moving stories closer to the source code as well.